desktop/e2e/tests/lib/video-recorder.js

```diff
@@ -12,8 +12,38 @@ const ffmpeg = require( 'ffmpeg-static' );
 
 const E2E_DIR = path.resolve( __dirname, '../../' );
 
+// Helper to get device/display identifier (Linux)
+function getFreeDisplay() {
+	let i = 0;
+	while ( ! fs.existsSync( `/tmp/.X${ i }-lock` ) && i < 200 ) {
```

Comment:
On Linux, display identifier should reasonably be between 0 and 200 (it's arbitrarily set to `99` in the Docker instances, but this could vary. Capped at 200 to avoid an infinite loop). If a correct device id isn't located, `ffmpeg` recording will simply silently fail (and won't affect e2e test results) 👍.